From f52b7d10369eb0a2bb79b6dbfa6151df94974094 Mon Sep 17 00:00:00 2001
From: Ian Dannapel <ian.dannapel@iris-sensing.com>
Date: Tue, 24 May 2022 16:02:25 +0200
Subject: [PATCH 33/33] imx8mp-irma6r2.dts: Add csi_mclk to pinctrl

This drives the Pin connected to EXPO_TRIG from EPC660 to low.
According to the Datasheet_epc660-V2.15:
If HW shutter is not used, connect this pin to GND

Signed-off-by: Ian Dannapel <ian.dannapel@iris-sensing.com>
---
 arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts b/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
index 510388ce434d..fc19e92c0757 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mp-irma6r2.dts
@@ -155,7 +155,7 @@
 		reg = <0x22>;
 		status = "okay";
 		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_csi0_rst>;
+		pinctrl-0 = <&pinctrl_csi0_rst>, <&pinctrl_csi_mclk>;
 		reset-gpios = <&gpio4 17 GPIO_ACTIVE_LOW>;
 
 		#address-cells = <1>;
@@ -713,6 +713,12 @@
 		>;
 	};
 
+	pinctrl_csi_mclk: csi_mclk_grp {
+		fsl,pins = <
+			MX8MP_IOMUXC_GPIO1_IO15__CCM_CLKO2	0x59
+		>;
+	};
+
 	pinctrl_pca9554: pca9554_grp {
 		fsl,pins = <
 			MX8MP_IOMUXC_SAI1_MCLK__GPIO4_IO20		0x19
-- 
2.25.1

